tests/app/clients/test_zendesk_sell.py

```diff
@@ -356,3 +361,16 @@ def test_send_create_service(
         assert ZenDeskSell().send_create_service(sample_service, sample_service.users[0])
         upsert_contact_mock.assert_called()
         upsert_deal_mock.assert_called()
+
+
+def test_send_go_live_service(
+        notify_api: Flask,
+        sample_service: Service,
+        mocker: MockFixture):
+
+    upsert_contact_mock = mocker.patch('app.user.rest.ZenDeskSell.upsert_contact', return_value=(1, True))
+    upsert_deal_mock = mocker.patch('app.user.rest.ZenDeskSell.upsert_deal', return_value=1)
+    with notify_api.app_context():
+        assert ZenDeskSell().send_go_live_service(sample_service, sample_service.users[0])
+        upsert_contact_mock.assert_called()
+        upsert_deal_mock.assert_called()
```

Comment:
You've got tests covering those already, could you make sure that they're called with the appropriate parameters?